qme/app/server.py

```diff
@@ -39,7 +40,7 @@ def start(port=5000, debug=True, queue=None, host=None):
        the hostname, set the environment variable QME_HOSTNAME or set on command
        line with qme start.
     """
-    bot.info("QueueMe!")
+    bot.info(f"QueueMe: running on http://localhost:{port}")
```

Comment:
There is host variable and then

host=host or QME_HOSTNAME

what is it's role? Could be something else than localhost?

Reply:
yes.

Reply:
But then why localhost hardcoded in the message? Shouldn't it user that host variable?

Reply:
Was this the one I missed, just from an hour ago? So sorry! I didn't realize you were saying to add host to the rendered string, it's totally logical and I just updated the PR!

And to better answer your question, the need for this variable is when we create a container - if you run a Dockerized version and use port 127.0.0.1 (same as localhost) it won't be broadcast to the local machine. Instead you need 0.0.0.0, so that's why it's a variable there.